swig/perl/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 include(UseSWIG)
 
 set(CMAKE_SWIG_FLAGS "-module" "openscap_pm")
+if (${CMAKE_C_COMPILER_ID} STREQUAL "GNU" OR ${CMAKE_C_COMPILER_ID} STREQUAL "Clang")
+	set_property(SOURCE openscapPERL5_wrap.c PROPERTY COMPILE_OPTIONS "-w")
```

Comment:
This looks fine as it is supported by VSCC: https://msdn.microsoft.com/en-us/library/thxezb7y.aspx and clang: https://clang.llvm.org/docs/UsersManual.html